packages/admin-ui-hooks/src/test-utils.tsx

```diff
@@ -0,0 +1,72 @@
+import 'jest-location-mock'
+
+export function jestWindowMock() {
```

Comment:
Why do you need this implementation? Would be nice an comment on `jestWindowMock` to explain which kind of behavior you expect on using it that the original one doesn't solve.

Reply:
Yes, could be. Just would be nice an explanation why jest-location-mock or native `window.location` isn't enough for your use case.